.github/workflows/pr.yaml

```diff
@@ -8,10 +8,10 @@ on:
 jobs:
   shared-operator-workflow:
     name: shared-operator-workflow
-    uses: redhat-cop/github-workflows-operators/.github/workflows/pr-operator.yml@v1.0.4
+    uses: garethahealy/github-workflows-operators/.github/workflows/pr-operator.yml@main
     with: 
       RUN_UNIT_TESTS: true
       RUN_INTEGRATION_TESTS: false
       RUN_HELMCHART_TEST: true
-      GO_VERSION: ~1.18
+      GO_VERSION: ~1.19
```

Comment:
needed to bump version, as CI was failing. see:
- https://github.com/garethahealy/patch-operator/actions/runs/6890011125/job/18742197862